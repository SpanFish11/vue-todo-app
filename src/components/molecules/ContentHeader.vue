<script setup></script>

<template>
  <div class="d-flex justify-content-between align-items-center mb-4">
    <h1 class="fw-bold text-white">T O D O</h1>
    <img alt="moon" src="../../assets/icons/moon-fill.svg" class="img" />
  </div>
</template>

<style scoped lang="scss">
.img {
  filter: invert(100%) sepia(0%) saturate(0%) hue-rotate(0deg) brightness(100%) contrast(100%);
  width: 25px;
  height: 25px;
}
</style>
