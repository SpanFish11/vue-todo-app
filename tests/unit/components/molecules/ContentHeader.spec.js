import { describe, expect, it } from '@jest/globals'
import { shallowMount } from '@vue/test-utils'
import ContentHeader from '@/components/molecules/ContentHeader.vue'

describe('Given the ContentHeader component', () => {
  it('should render', () => {
    const wrapper = shallowMount(ContentHeader, {})

    expect(wrapper.element).toMatchSnapshot()
  })
})
