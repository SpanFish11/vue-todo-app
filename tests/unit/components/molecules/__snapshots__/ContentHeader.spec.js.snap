// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given the ContentHeader component should render 1`] = `
<div
  class="d-flex justify-content-between align-items-center mb-4"
>
  <h1
    class="fw-bold text-white"
  >
    T O D O
  </h1>
  <img
    alt="moon"
    class="img"
    src=""
  />
</div>
`;
